migrate/v0_13/migrate.go

```diff
@@ -78,12 +90,232 @@ func MigrateCDP(oldGenState v0_11cdp.GenesisState) v0_13cdp.GenesisState {
 	)
 }
 
+// MigrateHard migrates from a v0.11 hard (harvest) genesis state to a v0.13 hard genesis state
+func MigrateHard(genesisState v0_11hard.GenesisState) v0_13hard.GenesisState {
+	v13Deposits := v0_13hard.Deposits{}
+	v13DepositorMap := make(map[string]v0_13hard.Deposit)
+	v13GenesisAccumulationTimes := v0_13hard.GenesisAccumulationTimes{}
+	v13TotalSupplied := sdk.NewCoins()
+
+	for _, dep := range genesisState.Deposits {
+		v13Deposit, ok := v13DepositorMap[dep.Depositor.String()]
+		if !ok {
+			v13Deposit := v0_13hard.NewDeposit(dep.Depositor, sdk.NewCoins(dep.Amount), v0_13hard.SupplyInterestFactors{v0_13hard.NewSupplyInterestFactor(dep.Amount.Denom, sdk.OneDec())})
+			v13DepositorMap[dep.Depositor.String()] = v13Deposit
+		} else {
+			v13Deposit.Amount = v13Deposit.Amount.Add(dep.Amount)
+			v13Deposit.Index = append(v13Deposit.Index, v0_13hard.NewSupplyInterestFactor(dep.Amount.Denom, sdk.OneDec()))
+			v13DepositorMap[dep.Depositor.String()] = v13Deposit
+		}
+	}
+
+	defaultInterestModel := v0_13hard.NewInterestRateModel(sdk.ZeroDec(), sdk.MustNewDecFromStr("0.05"), sdk.MustNewDecFromStr("0.8"), sdk.MustNewDecFromStr("1.0"))
+	newParams := v0_13hard.NewParams(
+		v0_13hard.MoneyMarkets{
+
+			v0_13hard.NewMoneyMarket("btcb", v0_13hard.NewBorrowLimit(true, sdk.ZeroDec(), sdk.MustNewDecFromStr("0.5")), "btc:usd", sdk.NewInt(100000000),
+				defaultInterestModel,
+				sdk.MustNewDecFromStr("0.025"), sdk.MustNewDecFromStr("0.02"),
+			),
+			// xrpb
+			v0_13hard.NewMoneyMarket("xrpb", v0_13hard.NewBorrowLimit(true, sdk.ZeroDec(), sdk.MustNewDecFromStr("0.5")), "xrp:usd", sdk.NewInt(100000000),
+				defaultInterestModel,
+				sdk.MustNewDecFromStr("0.025"), sdk.MustNewDecFromStr("0.02"),
+			),
+			// busd
+			v0_13hard.NewMoneyMarket("busd", v0_13hard.NewBorrowLimit(true, sdk.MustNewDecFromStr("100000000000000"), sdk.MustNewDecFromStr("0.5")), "busd:usd", sdk.NewInt(100000000),
+				defaultInterestModel,
+				sdk.MustNewDecFromStr("0.025"), sdk.MustNewDecFromStr("0.02"),
+			),
+			// usdx
+			v0_13hard.NewMoneyMarket("usdx", v0_13hard.NewBorrowLimit(true, sdk.ZeroDec(), sdk.ZeroDec()), "usdx:usd", sdk.NewInt(1000000),
```

Comment:
why is usdx LTV 0?

Reply:
It doesn't count towards borrowing at launch. Ie, users can't borrow using USDX collateral. This is until the pricefeed and stability fixes for USDX go into effect. We'll raise the LTV gradually after launch.